### ISSUE TITLE
Updates in Huawei AppGallery 

### ISSUE THREAD

**Issue by icono-stas** [radimitrov/CSharpShellApp#400]
Current app version in AppGallery is 3.2.54 and it has some freezing issues which are fixed in later versions. I know it's annoying to push updates to alternative app stores for like 2 or 3 people who use them, but sadly app purchases through Play Market are still unavailable in many countries. Maybe you could add a way to move app purchases from AppGallery to Play Market installations via some token? I'm even fine with buying the app again, it's worth every single cent, but it's pretty difficult to do in my country.

**Comment by radimitrov** [comment]
You're right, I will update the Huawei version. Should be live in about a day.
As for cross store purchase synchronization, I can't do that since it violates Google/Huawei's policy rules. 
And Google is already a massive annoyance for policies. Did you know this IDE app doesn't require full storage access option? Everything can be done via storage API interop copying for your convenience... They allowed the new update yesterday so the solution was to remove the ability to use the Android Scoped Storage option in the file manager as they implied. Yeah... Stubit...

**Comment by icono-stas** [comment]
Google being Google again, pretty sure they'll ruin every service they have if they keep following this direction. Thanks for the good work though, I can imagine how annoying it is to push updates to multiple stores with your update frequency. I wish I could donate, but PayPal isn't an option here either
